### PR TITLE
add id80/ansi/keymaps/msf

### PR DESCRIPTION
**Pull request by msf**
This adds my personal keymap for the ansi variant of the id80 keyboard.

It's just a keymap submission :-)
I have it running on my own id80, compiles and flashes without issues.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (**addition** or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/id80/ansi/keymaps/msf/config.h [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/*
```
Comment (drashna) [inline keyboards/id80/ansi/keymaps/msf/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/id80/ansi/keymaps/msf/config.h (hunk 1 above)
Comment (msf) [reply keyboards/id80/ansi/keymaps/msf/config.h:17]: done. thanks

**Comment by msf** [comment]
I've now addressed the comments. Please review again when possible. thanks for your patience

**Review by tzarc — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**